README.md

```diff
@@ -155,6 +57,12 @@ Use "picotool help <cmd>" for more info
 
 Note commands that aren't acting on files require a device in BOOTSEL mode to be connected.
 
+## Building & Installing
+
+If you don't want to build picotool yourself, you can find pre-built executables for Windows, macOS, and Linux in the [pico-sdk-tools](https://github.com/raspberrypi/pico-sdk-tools/releases) repository. Assuming you've extracted that archive to `$EXTRACT_LOCATION` (with the actual picotool executable at `$EXTRACT_LOCATION/picotool/picotool`), you can point the Pico SDK at this binary by setting the `picotool_DIR` environment variable to `$EXTRACT_LOCATION/picotool`, or by passing `-Dpicotool_DIR=$EXTRACT_LOCATION/picotool` to your `cmake` command or setting it in your `CMakeLists.txt` file.
```

Comment:
Reading this again, would using `<extract_location>` to signify a placeholder-directory work better than using `$EXTRACT_LOCATION`? Because otherwise some users might think that they need to set an `$EXTRACT_LOCATION` environment variable? :thinking:

Reply:
Yeah, that's clearer
```suggestion
If you don't want to build picotool yourself, you can find pre-built executables for Windows, macOS, and Linux in the [pico-sdk-tools](https://github.com/raspberrypi/pico-sdk-tools/releases) repository. Assuming you've extracted that archive to `<extract_location>` (with the actual picotool executable at `<extract_location>/picotool/picotool`), you can point the Pico SDK at this binary by setting the `picotool_DIR` environment variable to `<extract_location>/picotool`, or by passing `-Dpicotool_DIR=<extract_location>/picotool` to your `cmake` command or setting it in your `CMakeLists.txt` file.
```